packages/client/src/utils/fetchApi.tsx

```diff
@@ -1,83 +1,72 @@
-import baseApiConfigConnection from '../constants/baseApiConfigConnection'
-
 const enum METHODS {
   GET = 'GET',
   POST = 'POST',
   PUT = 'PUT',
   DELETE = 'DELETE',
+  PATCH = 'PATCH',
 }
 
-type Options<T> = {
-  body?: T | undefined
-  isFormData?: boolean
-} & Omit<RequestInit, 'body'>
-
-type Request<T> = (url: string, options?: Options<T>) => Promise<Response>
-
-const configOptions = {
-  method: METHODS.GET,
-  credentials: 'include' as RequestCredentials | undefined,
-  headers: baseApiConfigConnection.headers,
+type Options = {
+  body?: Record<string, unknown> | FormData
 }
 
-export class FetchApi {
-  static API_URL = baseApiConfigConnection.url
+type Request = <T>(url: string, options?: Options) => Promise<T>
 
-  getApiUrl = () => {
-    return FetchApi.API_URL
-  }
+export class FetchApi {
+  static API_URL = '/api/v2'
 
-  private buildUrl = (path: string) => {
-    return FetchApi.API_URL + path
+  public get: Request = async (url: string) => {
+    return await baseFetch(url, METHODS.GET)
   }
 
-  public get: Request<undefined> = async (url: string, options = {}) => {
-    const buildedUrl = this.buildUrl(url)
-    const res = await fetch(buildedUrl, {
-      ...options,
-      ...configOptions,
-    })
-    const result = await res.json()
-    if (res.status !== 200 && res.status !== 304) {
-      return Promise.reject(result.reason)
-    }
-    return result
+  public post: Request = async (url: string, options = {}) => {
+    return await baseFetch(url, METHODS.POST, options.body)
```

Comment:
Нет необходимости написать return await, достаточно проще:
```suggestion
  public post: Request = async (url: string, options = {}) => {
    return baseFetch(url, METHODS.POST, options.body)
```
Подробнее можешь тут прочитать https://eslint.org/docs/latest/rules/no-return-await